### DISCUSSION TITLE
Postgres backup file always empty

### DISCUSSION THREAD

**Discussion by bolenton** [Dokploy/dokploy#1014] [Q&A]
I am using the postgres Template and I have set up the automated backup to my s3 endpoint (Hetzner Buckets). The postgres database is Live and has data, The s3 connection is successful and the backup runs every day. However the backup SQL file in my bucket is always 20bytes and when I open it there is no data.

TLDR: my automated Postgres backup file is created every day, however it is always empty.

**Comment by Siumauricio** [comment]
Can you add more details to reproduce the issue?

**Reply by bolenton** [reply]
Not too sure what to add, I basically just following the steps to add backups using the UI according to the official docs and the backup seems to work except there is no data. Maybe I'm missing something? Do I have to enable something on my database side?
